docs/topics/i18n/translation.txt

```diff
@@ -1431,10 +1446,17 @@ Setting ``prefix_default_language`` to ``False`` removes the prefix from the
 default language (:setting:`LANGUAGE_CODE`). This can be useful when adding
 translations to existing site so that the current URLs won't change.
 
+When using :func:`~django.conf.urls.i18n.i18n_patterns` within
+:setting:`ROOT_URLCONF`, be aware that any JavaScript catalog routes will also
+need to be placed within your :func:`~django.conf.urls.i18n.i18n_patterns`
```

Comment:
I'd replace this phrase with ".. any Javascript catalog router will also need to be wrapped by :func:`~django.conf.urls.i18n.i18n_patterns in order..."